### PR TITLE
Show page number only when toolbar is hidden

### PR DESCRIPTION
**Pull request by onlymash**
The toolbar or navigation bar is translucent, so the page number should be hidden

### CODE REVIEW

File: app/src/main/java/eu/kanade/tachiyomi/ui/reader/ReaderActivity.kt [hunk 1]
```diff
@@ -37,12 +37,10 @@ import eu.kanade.tachiyomi.util.lang.plusAssign
...
 import eu.kanade.tachiyomi.util.system.GLUtil
 import eu.kanade.tachiyomi.util.system.toast
-import eu.kanade.tachiyomi.util.view.gone
-import eu.kanade.tachiyomi.util.view.visible
+import eu.kanade.tachiyomi.util.view.*
 import eu.kanade.tachiyomi.widget.SimpleAnimationListener
 import eu.kanade.tachiyomi.widget.SimpleSeekBarListener
 import kotlinx.android.synthetic.main.reader_activity.*
-import me.zhanghai.android.systemuihelper.SystemUiHelper
 import nucleus.factory.RequiresPresenter
```
Comment (arkon) [inline app/src/main/java/eu/kanade/tachiyomi/ui/reader/ReaderActivity.kt]: I believe you can remove a dependency now.

File: app/src/main/java/eu/kanade/tachiyomi/ui/reader/ReaderActivity.kt [hunk 2]
```diff
@@ -37,12 +37,10 @@ import eu.kanade.tachiyomi.util.lang.plusAssign
 import eu.kanade.tachiyomi.util.storage.getUriCompat
 import eu.kanade.tachiyomi.util.system.GLUtil
 import eu.kanade.tachiyomi.util.system.toast
-import eu.kanade.tachiyomi.util.view.gone
-import eu.kanade.tachiyomi.util.view.visible
+import eu.kanade.tachiyomi.util.view.*
 import eu.kanade.tachiyomi.widget.SimpleAnimationListener
 import eu.kanade.tachiyomi.widget.SimpleSeekBarListener
 import kotlinx.android.synthetic.main.reader_activity.*
-import me.zhanghai.android.systemuihelper.SystemUiHelper
```
Comment (arkon) [inline app/src/main/java/eu/kanade/tachiyomi/ui/reader/ReaderActivity.kt]: I believe you can remove a dependency now.

File: app/src/main/java/eu/kanade/tachiyomi/ui/reader/ReaderActivity.kt [hunk 3]
```diff
@@ -570,9 +579,6 @@ class ReaderActivity : BaseRxActivity<ReaderPresenter>() {
             subscriptions += preferences.trueColor().asObservable()
                     .subscribe { setTrueColor(it) }
 
-            subscriptions += preferences.fullscreen().asObservable()
```
Comment (arkon) [inline app/src/main/java/eu/kanade/tachiyomi/ui/reader/ReaderActivity.kt]: Shouldn't we still have something like this to handle it when it gets toggled in the reader?

Comment (onlymash) [reply app/src/main/java/eu/kanade/tachiyomi/ui/reader/ReaderActivity.kt]: setMenuVisibility will be called

**Comment by arkon** [comment]
Tested in API 21, and it looks like this after toggling fullscreen off:

Screen Shot 2020-02-18 at 17 29 22

It's fine after the activity gets recreated though (i.e. exiting and entering again or rotating).

File: app/src/main/res/layout/reader_activity.xml [hunk 4]
```diff
@@ -84,9 +83,7 @@
             <eu.kanade.tachiyomi.ui.reader.ReaderSeekBar
                 android:id="@+id/page_seekbar"
                 android:layout_width="0dp"
-                android:layout_height="wrap_content"
-                android:maxHeight="?attr/actionBarSize"
-                android:minHeight="?attr/actionBarSize"
```
Comment (arkon) [inline app/src/main/res/layout/reader_activity.xml]: Can you double check that this doesn't cause a regression for #2487?

File: app/src/main/java/eu/kanade/tachiyomi/ui/reader/ReaderActivity.kt [hunk 5]
```diff
@@ -278,7 +283,12 @@ class ReaderActivity : BaseRxActivity<ReaderPresenter>() {
     private fun setMenuVisibility(visible: Boolean, animate: Boolean = true) {
         menuVisible = visible
         if (visible) {
-            systemUi?.show()
+            if (preferences.fullscreen().getOrDefault()) {
+                window.showBar()
+            } else {
+                reader_menu.setPadding(0, 0, 0, 0)
+                window.defaultBar()
+            }
```
Comment (arkon) [inline app/src/main/java/eu/kanade/tachiyomi/ui/reader/ReaderActivity.kt]: Might be nice to extract this into a separate function.

**Pull request merged**